common-react/@dbeaver/ui-kit/src/Button/_base.css

```diff
@@ -0,0 +1,21 @@
+@import 'tailwindcss';
+
+@layer base {
+  :root {
+    --btn-height: var(--control-height-medium);
+    --btn-padding-inline: --spacing(2.5);
+    --btn-gap: calc(var(--btn-padding-inline) / 4);
+    --btn-foreground: var(--color-white);
+    --btn-background: var(--color-primary-600);
+    --btn-background-hover: var(--color-primary-700);
+    --btn-background-active: var(--color-primary-800);
+    --btn-border-width: 1px;
+    --btn-border-color: transparent;
+    --btn-border-style: solid;
+    --btn-radius: var(--radius-md);
```

Comment:
i think we need some base variables on the theme layout for global things like:
1. controls radius - it may be used for buttons, inputs, text areas (examples)
2. maybe padding (like spacing between elements)

Reply:
I added vars for border, but I'm not sure about spacing right now. First of all, we can use tailwind directly, they have spacing units like p(padding) and m(margin) based on rem, they have container units like w- and h- based on rem.
Plus, creating the layout and grid units is more about product, not the UI-kit itself until it's very opinionated and have to be used as is which is not true in our case. We can create components / recommendation/ guidelines of how to use spacing **in the product**.